Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionContext.mm

```diff
@@ -541,7 +544,12 @@ - (BOOL)hasContentModificationRules
 
 - (void)loadBackgroundContentWithCompletionHandler:(void (^)(NSError *error))completionHandler
 {
-    Ref { *_webExtensionContext }->loadBackgroundContent(makeBlockPtr(completionHandler));
+    Ref { *_webExtensionContext }->loadBackgroundContent([&](RefPtr<API::Error> outError) {
```

Comment:
`outError` is wrong here since it is passed _in_, just `error` is fine.

```suggestion
    Ref { *_webExtensionContext }->loadBackgroundContent([&](RefPtr<API::Error> error) {
```